portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetSorting.testcase

```diff
@@ -25,57 +53,103 @@ definition {
-	@ignore = "true"
 	@priority = 4
 	test CanAssertSortingIsRequired {
+		task ("And adds a new default sort, on "+" button or "New Default Sort" button") {
+			LexiconEntry.gotoAdd();
+		}
 
-		// TODO LPS-185325 CanAssertSortingIsRequired pending implementation
+		task ("And the new default sort modal is displayed") {
+			AssertElementPresent(
+				locator1 = "Portlet#MODAL_TITLE",
+				value1 = "New Default Sort");
+		}
 
+		task ("Then Sorting is required") {
+			AssertElementPresent(
+				key_text = "Sorting",
+				locator1 = "RaylifeApplicationModal#MODAL_DESCRIPTION");
+		}
 	}
 
 	@description = "LPS-185326. Confirm that the Default Sort was created successfully."
-	@ignore = "true"
 	@priority = 3
 	test CanCreateDefaultSort {
+		task ("And adding a new default sort ") {
+			FrontendDataSetAdmin.createASort(
+				key_field = "creator",
+				key_sorting = "Descending");
+		}
 
-		// TODO LPS-185326 CanCreateDefaultSort pending implementation
-
-	}
-
-	@description = "LPS-185328. Confirm that the dataset and view created via the API appears when searched."
-	@ignore = "true"
-	@priority = 4
-	test CanCreateDefaultSortViaAPI {
-
-		// TODO LPS-185328 CanCreateDefaultSortViaAPI pending implementation
-
+		task ("Then the success message will be the Liferay generic message") {
+			AssertTextEquals.assertPartialText(
+				locator1 = "Message#SUCCESS",
+				value1 = "Your request completed successfully.");
+		}
 	}
 
 	@description = "LPS-185322. Confirm that the new default sorting modal is displayed."
-	@ignore = "true"
 	@priority = 4
 	test CanDisplayedDefaultSort {
+		task ("And adds a new default sort, on "+" button or "New Default Sort" button.") {
+			LexiconEntry.gotoAdd();
+		}
 
-		// TODO LPS-185322 CanDisplayedDefaultSort pending implementation
-
+		task ("And the new default sort modal is displayed") {
+			AssertElementPresent(
+				locator1 = "Portlet#MODAL_TITLE",
+				value1 = "New Default Sort");
+		}
 	}
 
 	@description = "LPS-185327. Confirm that the search bar responds when we search for one of the options."
-	@ignore = "true"
 	@priority = 4
 	test CanSearchSortingInSearchBar {
+		task ("And adding a new default sort/the fields are selected ") {
+			FrontendDataSetAdmin.createASort(
+				key_field = "creator",
+				key_sorting = "Descending");
+		}
 
-		// TODO LPS-185327 CanSearchSortingInSearchBar pending implementation
+		task ("And then the search bar responds when we search one of the options listed") {
+			Type(
+				locator1 = "AppBuilder#SEARCH_BAR_INPUT",
+				value1 = "creator");
 
+			AssertElementPresent(
+				key_name = "creator",
+				locator1 = "FrontendDataSetAdmin#ORDERABLE_TABLE");
+		}
+
+		task ("Then the search bar responds when we search for a non-existent option") {
+			Type(
+				locator1 = "AppBuilder#SEARCH_BAR_INPUT",
+				value1 = "any");
+
+			AssertTextEquals(
+				locator1 = "FrontendDataSetAdmin#TEXT_STATUS",
+				value1 = "No Results Found");
```

Comment:
```suggestion
			AssertTextEquals(
				locator1 = "Message#EMPTY_STATE_INFO",
				value1 = "No default sort created yet.");
```
Hey @andreluis2022 I believe you can use this path that already exists instead of creating another one